erpnext/www/lms/program.py

```diff
@@ -26,4 +27,13 @@ def get_program(program_name):
 
 def get_course_progress(courses, program):
 	progress = {course.name: utils.get_course_progress(course, program) for course in courses}
-	return progress
\ No newline at end of file
+	return progress
+
+def calculate_course_progress(course_progress):
+	total_courses = len(course_progress)
+	completed_courses = 0
+	for progress in course_progress:
+		if course_progress[progress].get("completed"):
```

Comment:
```suggestion
		if progress.get("completed"):
```

Reply:
We can't use **progress.get("completed")** instead of **course_progress[progress].get("completed")**

If I made the changes which you suggested then it will throw an error and we can't achieve the expected result.
To get accurate output we have to use **if course_progress[progress].get("completed"):**